dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/BundleUtil.java

```diff
@@ -123,23 +126,35 @@ private static String getStringFromInternalBundle(String bundleKey, String bundl
                     ? resourceBundle.getString(bundleKey)
                     : StringUtils.EMPTY;
         } catch (Exception ex) {
-            logger.warning("Could not find key \"" + bundleKey + "\" in bundle file: " + bundleName);
+            logger.finest("Could not find key \"" + bundleKey + "\" in bundle file: " + bundleName);
             return StringUtils.EMPTY;
         }
     }
 
-    private static Optional<String> getStringFromExternalBundle(String bundleKey, String bundleName, Locale locale) {
+    // NB this method is nearly exact copy of getStringFromInternalBundle(…), however
```

Comment:
what NB means here? Maybe consider making it a JavaDoc for readability reasons?

Reply:
NB = Nota Bene, however I've changed to IMPORTANT.
I'd rather not change that comment to javadoc, as it's only about code details.